addons/hr/models/res_users.py

```diff
@@ -295,3 +295,18 @@ def action_create_employee(self):
             company_id=self.env.company.id,
             **self.env['hr.employee']._sync_user(self)
         ))
+
+    def action_open_employees(self):
+        self.ensure_one()
+        employees = self.env["hr.employee"].search([('user_id', '=', self.id)])
+        if len(employees) > 1:
+            action = self.env.ref("hr.hr_employee_action_from_user").read()[0]
+            return action
```

Comment:
I think for consistency it will be better if we remove `hr_employee_action_from_user` action and replace with

```suggestion
            return {
                'name': _('Employee'),
                'type': 'ir.actions.act_window',
                'res_model': 'hr.employee',
                'view_mode': 'kanban,tree,form',
                'domain': [('id', 'in', self.employee_ids.ids)],
            }
```